docs/help/Resources-for-learning-our-technology-stack.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 1
+id: index
+---
+# Technology Stack Resources
+
+## Frontend 
+
+### React 
+* [React website](https://reactjs.org/) - provides some resources that are helpful for getting started.
+
+#### Books / Courses / Tutorials
+* [Epic React](https://epicreact.dev/) - paid course that is popular at Truss.
+
+### Truss Resources
+* [Truss engineering playbook frontend resources](https://transcom.github.io/mymove-docs/docs/frontend)
+* [MilMove Frontend docs](https://transcom.github.io/mymove-docs/docs/frontend)
+
+## Backend 
+
+### Go 
+* [Golang Weekly newsletter](https://golangweekly.com) - source for a large number of the resources linked below.
+
+#### Books / Courses / Tutorials
+* [A Tour of Go](https://go.dev/tour/list) - this is a nice entry point for the basics.
+* [Learn Go with Tests](https://quii.gitbook.io/learn-go-with-tests/)
+* [Practical Go Lessons](https://www.practical-go-lessons.com)
+* [Darker Corners of Go](https://rytisbiel.com/2021/03/06/darker-corners-of-go)
+* [Go By Example](https://gobyexample.com/)
+
```

Comment:
Just putting JP's comment as a suggestion so that it's easy to merge, if desired.

```suggestion
* [Go the Complete Developers Guide](https://www.udemy.com/course/go-the-complete-developers-guide)

```